### PR TITLE
[Refactor] Merge player update

### PR DESCRIPTION
**Pull request by Atralupus**
Solved https://github.com/planetarium/9c-launcher/pull/1834#pullrequestreview-1084704220

### CODE REVIEW

File: src/main/update/launcher-update.ts [hunk 1]
```diff
@@ -141,6 +111,36 @@ export async function update(update: Update, listeners: IUpdateOptions) {
   console.log("launcherDownloadUrl: ", launcherDownloadUrl);
   console.log("playerDownloadUrl: ", playerDownloadUrl);
 
+  if (peerVersionNumber <= localVersionNumber) {
```
Comment (BasixKOR) [inline src/main/update/launcher-update.ts:114]: Could you comment on the reason for this change? I can't quite grasp what difference moving this chunk make.

**Review by BasixKOR — COMMENTED** [review]
Pretty okay I think, but I have some questions.

File: src/main/update/launcher-update.ts (hunk 1 above)
Comment (Atralupus) [reply src/main/update/launcher-update.ts:114]: just... for creating `playerDownloadUrl` before creating lockfile

Comment (BasixKOR) [reply src/main/update/launcher-update.ts:114]: Ooh I see.

**Review by BasixKOR — APPROVED** [review]

**Comment by Atralupus** [comment]
🚀

**Pull request merged**